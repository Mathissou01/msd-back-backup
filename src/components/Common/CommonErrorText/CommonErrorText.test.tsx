import { render } from "@testing-library/react";
import CommonErrorText from "./CommonErrorText";

const mock = {
  message: "error message",
  errorId: "error-1",
};

it("renders", () => {
  const { container } = render(
    <CommonErrorText message={mock.message} errorId={mock.errorId} />,
  );
  expect(container).toMatchSnapshot();
});
