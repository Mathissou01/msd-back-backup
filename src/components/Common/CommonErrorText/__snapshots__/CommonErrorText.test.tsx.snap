// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <em
    class="c-CommonErrorMessage"
    id="error-1"
    role="alert"
  >
    error message
  </em>
</div>
`;
